com.ibm.wala.core.testdata/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -81,7 +81,7 @@ org.eclipse.jdt.core.compiler.problem.potentiallyUnclosedCloseable=error
 org.eclipse.jdt.core.compiler.problem.rawTypeReference=warning
 org.eclipse.jdt.core.compiler.problem.redundantNullAnnotation=error
 org.eclipse.jdt.core.compiler.problem.redundantNullCheck=error
-org.eclipse.jdt.core.compiler.problem.redundantSpecificationOfTypeArguments=warning
+org.eclipse.jdt.core.compiler.problem.redundantSpecificationOfTypeArguments=ignore
```

Comment:
I am fine with not fixing these warnings in a `testdata` project. But if the decision is being made to ignore those issues, then we should tell Eclipse to be silent about them. That is exactly what this change does: it reclassifies redundant type arguments as `ignore` instead of `warning`.